boards/arm/nrf5340_pca10095/nrf5340_pca10095_cpu1.dts

```diff
@@ -0,0 +1,149 @@
+/*
+ * Copyright (c) 2019 Nordic Semiconductor ASA
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+/dts-v1/;
+#include <nordic/nrf5340_cpu1_qkaa.dtsi>
+
+/ {
+	model = "Nordic PCA10095 Dev Kit Network";
+	compatible = "nordic,pca10095-dk","nordic,nrf5340-cpu1-qkaa",
+			 "nordic,nrf5340-cpu1";
+
+	chosen {
+		zephyr,console = &uart0;
+		zephyr,shell-uart = &uart0;
+		zephyr,uart-mcumgr = &uart0;
+		zephyr,sram = &sram1;
+		zephyr,flash = &flash1;
+		zephyr,code-partition = &slot0_partition;
+	};
+
+
+	leds {
+		compatible = "gpio-leds";
+		led0: led_0 {
+			gpios = <&gpio0 28 GPIO_INT_ACTIVE_LOW>;
+			label = "Green LED 0";
+		};
+		led1: led_1 {
+			gpios = <&gpio0 29 GPIO_INT_ACTIVE_LOW>;
+			label = "Green LED 1";
+		};
+		led2: led_2 {
+			gpios = <&gpio0 30 GPIO_INT_ACTIVE_LOW>;
+			label = "Green LED 2";
+		};
+		led3: led_3 {
+			gpios = <&gpio0 31 GPIO_INT_ACTIVE_LOW>;
+			label = "Green LED 3";
+		};
+
+		buttons {
+			compatible = "gpio-keys";
+			button0: button_0 {
+				gpios = <&gpio0 23 GPIO_PUD_PULL_UP>;
+				label = "Push button 1";
+			};
+			button1: button_1 {
+				gpios = <&gpio0 24 GPIO_PUD_PULL_UP>;
+				label = "Push button 2";
+			};
+			button2: button_2 {
+				gpios = <&gpio0 8 GPIO_PUD_PULL_UP>;
+				label = "Push button 3";
+			};
+			button3: button_3 {
+				gpios = <&gpio0 9 GPIO_PUD_PULL_UP>;
+				label = "Push button 4";
+			};
+		};
+	};
+
+	/* These aliases are provided for compatibility with samples */
+	aliases {
+		led0 = &led0;
+		led1 = &led1;
+		led2 = &led2;
+		led3 = &led3;
+		sw0 = &button0;
+		sw1 = &button1;
+		sw2 = &button2;
+		sw3 = &button3;
+	};
+};
+
+&gpiote {
+	status = "okay";
+};
+
+&gpio0 {
+	status = "okay";
+};
+
+&gpio1 {
+	status = "okay";
+};
+
+&ipc {
+	status = "okay";
```

Comment:
This is redundant, this property is already set to "okay" in [dts/arm/nordic/nrf5340_cpu1.dtsi](https://github.com/zephyrproject-rtos/zephyr/pull/20157/files#diff-031e9ff221884ea236023201f93a779eR144).

Reply:
Right. Will remove. (no harm, though)

Reply:
Will remove from cpu0 as well.

Reply:
(if it exists)